src/platform_win.cc

```diff
@@ -88,7 +88,11 @@ void SetCurrentThreadName(const std::string& thread_name) {
   __try {
     RaiseException(MS_VC_EXCEPTION, 0, sizeof(info) / sizeof(ULONG_PTR),
                    (ULONG_PTR*)&info);
+#ifdef _MSC_VER
   } __except (EXCEPTION_EXECUTE_HANDLER) {
+#else
+  } catch (...) {
```

Comment:
Are you sure a C++ catch is able to catch a Structured Exception raised by RaiseException? IIRC, this does not work.

Reply:
No idea. As mentioned in #308 I've based this on https://github.com/EricssonResearch/cerbero/blob/master/recipes/webrtc-audio-processing/0004-Don-t-use-MSVC-specific-exception-handler-for-MINGW.patch